DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -55,6 +55,10 @@ constexpr float PADGRANULARITYTRKLSLOPE = 1000.f;
 constexpr float GRANULARITYTRKLPOS = 1.f / PADGRANULARITYTRKLPOS;     // granularity of position in tracklet64 word in pad-widths
 constexpr float GRANULARITYTRKLSLOPE = 1.f / PADGRANULARITYTRKLSLOPE; // granularity of slope in tracklet64 word in pads/timebin
 
+// Simulation specific parameters
+constexpr int TIMEBINSGARFIELD = 38; // Number of bins in time direction used for garfield simulation
+constexpr int ZBINSGARFIELD = 11;    // Number of bins in z direction used for garfield simulation
+
```

Comment:
I am happy with having all the constants in one place, particularly those used in several other units. Still, a few like these two are used only in a couple of places, for instance, in the definition of `Garfield` and in the drift estimators. So, any change in the constants will lead to recompiling and more classes that don't need to be recompiled. Therefore, I think `Garfield` can keep these constants for itself. Also, it's unlikely that these will change.

Reply:
I agree, that is changed now